src/constants/chainInfo.ts

```diff
@@ -167,4 +169,28 @@ export const CHAIN_INFO: ChainInfoMap = {
     logoUrl: polygonMaticLogo,
     nativeCurrency: { name: 'Polygon Mumbai Matic', symbol: 'mMATIC', decimals: 18 },
   },
+  [SupportedChainId.CELO]: {
+    networkType: NetworkType.L1,
+    blockWaitMsBeforeWarning: ms`10m`,
+    bridge: 'https://optics.app/',
+    docs: 'https://docs.celo.org/',
+    explorer: 'https://explorer.celo.org/',
+    infoLink: 'https://info.uniswap.org/#/celo/',
```

Comment:
Is there another pr in progress to add support for this? If not, we should probably drop the menu entry for celo instead of including a dead link.

Reply:
Yup! [just created it](https://github.com/Uniswap/v3-info/pull/235) 

There is one blocker I have that I've listed regarding the additional subgraph URI.

Reply:
Waiting for my teammate to deploy the remaining subgraph; going to remove the path to Celo for now.

Reply:
Removed the 'Celo' portion until the v3-info PR is deployed.

I'll create a separate PR to add the right link in case this one is merged first